test/FunctionalTests/Infrastructure/GrpcTestFixture.cs

```diff
@@ -74,5 +77,17 @@ public void Dispose()
             Client.Dispose();
             _server.Dispose();
         }
+
+        public void AssertTrailerStatus(StatusCode statusCode, string details)
+        {
+            Assert.AreEqual(statusCode.ToTrailerString(), TrailersContainer.Trailers[GrpcProtocolConstants.StatusTrailer].ToString());
```

Comment:
Nice. Now there is a central method for comparing the status code you can fix something that has been annoying me.

Update this AreEqual to include a message for when they aren't equal. Right now you get a message like "1 does not equal 13". Convert the trailer value back to the enum - `(StatusCode)Convert.ToInt32(trailerValue)` - and have a message like "Expected grpc-status Ok but got InternalError".